src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor

```diff
@@ -88,8 +89,8 @@
                 if (navLinkItem.ForceAnchor)
                 {
                     <a class="@GetLinkClass(navLinkItem)"
+                       style="padding: 0px 20px 0px @(((navLinkItem.Depth * 14) + 27).ToString())px;"
```

Comment:
please remove the px unit where the value of the CSS property is equal to 0.

Reply:
done